### PR TITLE
It begins

### PR DESCRIPTION
**Pull request by projectweekend**

### CODE REVIEW

File: pg_table_markdown/queries.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+                    data_type,
+                    character_maximum_length
+        FROM 		information_schema.columns
+        WHERE 		table_schema = '{0}'
+        ORDER BY    table_name
+    )
+    SELECT      JSON_AGG(table_schema_info.*)
+    FROM        table_schema_info;
+    """.format(table_schema)
+    return schema_query
```
Comment (foresmac) [inline pg_table_markdown/queries.py:17]: Should this file use tabs or convert to spaces?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+nose==1.3.6
```
Comment (foresmac) [inline requirements.txt:1]: Click should be in here, right?

Comment (foresmac) [inline requirements.txt:1]: Oh, wait, you're using the setup it handle getting click and psycopg2.

Comment (projectweekend) [inline requirements.txt:1]: Yeah, just put nose here for running tests... I guess I could just install it in the `.drone.yml` and the `Dockerfile` for local use. Thoughts?

**Comment by foresmac** [comment]
LGTM

**Pull request merged**